models/core/blocks.sql

```diff
@@ -1,4 +1,10 @@
-{{ config(materialized='incremental', unique_key='block_id', tags=['core']) }}
+{{ 
+    config(
+        materialized='incremental', 
+        unique_key='block_id', 
+        tags=['core']
```

Comment:
Might want to cluster_by block_timestamp? Maybe snowflake does it for us automatically tho?

Reply:
oh i actually didn't know this before but rather order the table by block_timestamp in the last line.
I think they do the same thing: https://docs.getdbt.com/reference/resource-configs/snowflake-configs
but happy to update specifying `cluster_by` in the config. what do you think?

Reply:
Sounds good. I've just been crawling flipside's pull requests and they all seem to use cluster_by. Might be good to follow them for now since they've been doing the same things for longer.